### DISCUSSION TITLE
Will this work - GoPro 8

### DISCUSSION THREAD

**Discussion by Videocamguy** [gopro/labs#422] [General]
>23:50!00:10NmNPp.120!S!23:45E!00:10R

**Comment by dnewman-gpsw** [comment]
should do.

**Reply by Videocamguy** [reply]
Thanks David.

**Comment by Videocamguy** [comment]
I fine tuned it a bit.. still good...

    >23:00!05:00NmNPp.300!S!22:50E!05:00R
